core/src/main/java/org/mapfish/print/servlet/oldapi/OldAPIMapPrinterServlet.java

```diff
@@ -223,18 +223,20 @@ public final void getFile(@PathVariable final String id,
      * @param baseUrl  the path to the webapp
      * @param jsonpVar if given the result is returned as a variable assignment
      * @param req      the http request
-     * @param resp     the http response
+     * @param appId    the app request
+     * @param resp     the http response     
```

Comment:
Yes, it is not namely for image layer. This is bug fix as well. I think somebody just forget add this argument for old api compabilites.